randomizer/Lists/FairyLocations.py

```diff
@@ -434,7 +434,7 @@ def __init__(
             region=Regions.BeyondHatch,
             fence=Fence(1820, 526, 2079, 874),
             spawn_y=60,
-            logic=lambda l: l.camera and l.punch and l.chunky,
+            logic=lambda l: l.camera and (l.punch and l.chunky) or l.phasewalk,
```

Comment:
needs more (). Right now, it's a bit unclear if you can get that fairy without camera.